From eb84443cc949a77daabc9333fc903fad87ca01cd Mon Sep 17 00:00:00 2001
From: Kevin Cernekee <cernekee@gmail.com>
Date: Mon, 18 Apr 2016 22:04:22 -0700
Subject: [PATCH 8/8] Fix missing -llz4 in static builds

List liblz4 under Requires.private in the pkg-config file.

Signed-off-by: Kevin Cernekee <cernekee@gmail.com>
---
 configure.ac      | 7 +++----
 openconnect.pc.in | 2 +-
 2 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 7da44dc..0851f9f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -577,11 +577,10 @@ AC_ARG_WITH(lz4,
   test_for_lz4=$withval,
   test_for_lz4=yes)
 
-enable_lz4=no
-if test "$test_for_lz4" = yes;then
+if test "$test_for_lz4" = yes; then
 PKG_CHECK_MODULES([LIBLZ4], [liblz4], [
-enable_lz4=yes
-AC_DEFINE([HAVE_LZ4], [], [LZ4 was found])
+	AC_SUBST(LIBLZ4_PC, liblz4)
+	AC_DEFINE([HAVE_LZ4], [], [LZ4 was found])
 ],
 [
 	AC_MSG_WARN([[
diff --git a/openconnect.pc.in b/openconnect.pc.in
index 53449df..8e295c4 100644
--- a/openconnect.pc.in
+++ b/openconnect.pc.in
@@ -7,7 +7,7 @@ includedir=@includedir@
 Name: openconnect
 Description: OpenConnect VPN client
 Version: @VERSION@
-Requires.private: @LIBPROXY_PC@ @ZLIB_PC@ @SSL_DTLS_PC@ @P11KIT_PC@ @LIBSTOKEN_PC@ @LIBPSKC_PC@ @LIBPCSCLITE_PC@ libxml-2.0
+Requires.private: @LIBPROXY_PC@ @ZLIB_PC@ @LIBLZ4_PC@ @SSL_DTLS_PC@ @P11KIT_PC@ @LIBSTOKEN_PC@ @LIBPSKC_PC@ @LIBPCSCLITE_PC@ libxml-2.0
 Libs: -L${libdir} -lopenconnect
 Libs.private: @INTL_LIBS@
 Cflags: -I${includedir}
-- 
1.9.1

